.github/workflows/main.yml

```diff
@@ -672,12 +677,20 @@ jobs:
 
       - name: Create Virtual Env for Tests
         uses: ./.github/actions/create-venv-for-tests
-        if: matrix.test-suite == 'integration' || matrix.test-suite == 'notebook' || matrix.test-suite == 'nonRawNotebook' || matrix.test-suite == 'remoteNotebook'
+        if: matrix.test-suite == 'integration' || matrix.test-suite == 'notebook' || matrix.test-suite == 'nonRawNotebook' || matrix.test-suite == 'remoteNotebook' || matrix.test-suite == 'notebookAndWebview'
+
+      # Set the correct xvfb commands to run vscode tests
+      # https://code.visualstudio.com/api/working-with-extensions/continuous-integration
+      # Note that xvfb github action only runs through xvfb on linux, so only set the commands there
+      # as on windows / mac they would get directly passed to run, not xvfb-run
+      - name: Set xvfb parameters linux
```

Comment:
xvfb doesn't support passing in server parameters (there is actually an open PR on the github to support this). To get around this we now just pass in the xvfb parameters to the run command, but only on linux, because on windows / mac xvfb just directly forwards on the run command. .xvfbCommand evaluates to empty string in those cases. Credit to @rchiodo for doing this instead of having to create a new action.